src/osd/PGTransaction.h

```diff
@@ -208,12 +209,12 @@ class PGTransaction {
 	return match(
 	  left,
 	  [&](const BufferUpdate::Write &w) -> bool {
-	    auto r = boost::get<BufferUpdate::Write>(&right);
+	    auto r = std::get_if<BufferUpdate::Write>(&right);
 	    return r != nullptr && (w.fadvise_flags == r->fadvise_flags);
 	  },
 	  [&](const BufferUpdate::Zero &) -> bool {
-	    auto r = boost::get<BufferUpdate::Zero>(&right);
-	    return r != nullptr;
+	    auto is_zero = std::holds_alternative<BufferUpdate::Zero>(right);
```

Comment:
(taste issue) no need for the 'is_zero'. The hold_alternative is clear enough.

Reply:
i concur with you, Ronen. just wanted to keep the existing structure. will change accordingly.